dashboard/config/scripts/levels/Allthethings Java Lab 3.level

```diff
@@ -0,0 +1,308 @@
+<Javalab>
+  <config><![CDATA[{
+  "properties": {
+    "encrypted": "false",
+    "instructions_important": "false",
+    "teacher_markdown": "Teachers should see this extra info.",
+    "long_instructions": "This is a sample Java Lab Neighborhood level.",
+    "mini_rubric": "false",
+    "hide_share_and_remix": "false",
+    "csa_view_mode": "neighborhood",
+    "serialized_maze": [
```

Comment:
Does this need to be stringified? i.e. the formatting is different for Allthethings Java Lab 1.level

Reply:
it actually supports both. This is what you end up with if you clone an existing level with a serialized maze, and the parsing on Javalab supports either a string or an array.